kumascript/src/api/web.js

```diff
@@ -83,6 +83,27 @@ module.exports = {
         page.url + hrefhash
       }"${titleAttribute}${flawAttribute}>${content}</a>`;
     }
+    if (!href.toLowerCase().startsWith("/en-us/")) {
+      // Before flagging this as a broken-link flaw, see if it's possible to
+      // change it to the en-US URL instead.
+      const hrefSplit = href.split("/");
+      hrefSplit[1] = "en-US";
+      const enUSPage = this.info.getPageByURL(hrefSplit.join("/"));
+      if (enUSPage.url) {
+        // But it's still a flaw. Record it so that translators can write a
+        // translated document to "fill the hole".
+        flaw = this.env.recordNonFatalError(
+          "broken-link",
+          `${hrefpath} does not exist, fallback to ${enUSPage.url}`
+        );
+        flawAttribute = ` data-flaw-src="${util.htmlEscape(flaw.macroSource)}"`;
+        return (
+          '<a class="page-only-in-english" ' +
+          'title="Page currently only available in English" ' +
+          `href="${enUSPage.url}"${flawAttribute}>${content} <small>(en-US)</small></a>`
+        );
+      }
```

Comment:
```suggestion
        return (
          '<a class="page-only-in-english" ' +
          'title="Page currently only available in English" ' +
          `href="${enUSPage.url}"${flawAttribute}>${content} <span>(en-US)</span></a>`
        );
```

Then in `client/src/document/index.scss` add the following:

```
.page-only-in-english {
  span {
    font-size: $small-font-size;
  }
}
```

Reply:
I know, sometimes I want to change my mind about the use of these elements but, we would need CSS anyhow to ensure that the `small` element uses the `$small-font-size`. I am kinda undecided on that one.